glossary.yml

```diff
@@ -1738,6 +1757,15 @@
       A naming convention in which the parts of a name are separated with
       dashes, as in `first-second-third`.
 
+- slug: keyword_arguments
+  ref:
+    - named_argument
+    - variable_arguments
+  en:
+    term: "keyword arguments"
+    def: >
+      Extra arguments given to a function as key-value pairs.
```

Comment:
Are these only a feature of python? If so, it might be useful to mention them and also label that they appear in python help documentation as "kwargs" or "\*\*kwargs".

Reply:
Other languages support them as well in various forms.